src/main/java/com/monumental/models/Monument.java

```diff
@@ -36,17 +36,20 @@
     private String material;
 
     @Column(name = "lat")
-    private double lat;
+    private Double lat;
 
     @Column(name = "lon")
-    private double lon;
+    private Double lon;
```

Comment:
This change is to make the fields nullable, since `double` defaults to `0.0` instead of `null`